README.md

```diff
@@ -124,4 +124,15 @@ docker run -p 80:5000 \
     -d devops-course-starter:dev-v0.1
 ```
 
-Note: See section on Trello setup for details of how to obtain the key and token values and setup the .env file.
\ No newline at end of file
+Note: See section on Trello setup for details of how to obtain the key and token values and setup the .env file.
+
+### Documentation
+
+C4 diagrams have been proded for this application in the file c4model.drawio. These can be viewed at https://app.diagrams.net/
+
+Within this file the  UML diagrams where generated by running;
+```
+pyreverse.exe -p ToDoApp app.py item.py status.py trello_config.py trello_items.py view_model.py
```

Comment:
This requires installing pylint first, which isn't in your project dependencies.